azure-local/concepts/compare-windows-server.md

```diff
@@ -69,60 +69,65 @@ The following table compares the workloads and benefits of Azure Local and Windo
 
 | Attribute | Azure Local | Windows Server |
 | ------------- | ------------------- | ----------------------- |
-| Azure Kubernetes Service (AKS)| Yes | Yes |
+| Azure Kubernetes Service (AKS)| Yes | Yes, but sunset <sup>1</sup> |
 | Azure Arc-Enabled PaaS Services | Yes | Yes |
-| Windows Server 2022 Azure Edition | Yes | No|
+| Windows Server 2022 Azure Edition | Yes | No |
+| Windows Server 2025 Azure Edition | Yes | No |
+| Windows Server Management Features via Arc (Nov. 2024) | Yes | Yes |
 | Windows Server subscription add-on (Dec. 2021)| Yes | No |
-| Free Extended Security Updates (ESUs) for Windows Server and SQL 2008/R2 and 2012/R2 | Yes | No <sup>1</sup>|
+| Free Extended Security Updates (ESUs) for Windows Server and SQL 2008/R2 and 2012/R2 | Yes | No <sup>2</sup> |
 
- <sup>1</sup> Requires purchasing an Extended Security Updates (ESU) license key and manually applying it to every VM.
+ <sup>1</sup> [AKS on Windows Server End of Support 27th March 2027](https://learn.microsoft.com/en-us/azure/aks/aksarc/aks-windows-server-retirement?tabs=ws).<br>
```

Comment:
The link needs correction in terms of linking to the GitHub page, Sorry I always have issues to understand the relative path to these internal references.